zerver/lib/actions.py

```diff
@@ -1870,18 +1870,21 @@ def notify_reaction_update(user_profile: UserProfile, message: Message,
 
 def do_add_reaction(user_profile: UserProfile, message: Message,
                     emoji_name: str, emoji_code: str, reaction_type: str) -> None:
+    """Should be called while holding a select_for_update lock
```

Comment:
Not sure if that's very update, but I'd rather name the lock by its proper name here - `FOR UPDATE`